### ISSUE TITLE
[Bug]: create if not exists user1,user2,user3... part of users not exists  create failed

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#5325]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

create user if not exists revoke_user_1 identified by '12345678',revoke_user_2 identified by '12345678',revoke_user_3 identified by '12345678',revoke_user_4 identified by '12345678',revoke_user_5 identified by '12345678';
create user if not exists revoke_user_1 identified by '12345678',revoke_user_2 identified by '12345678',revoke_user_3 identified by '12345678',revoke_user_4 identified by '12345678',revoke_user_5 identified by '12345678',revoke_user_6 identified by '12345678';
select * from mo_user;
![image](https://user-images.githubusercontent.com/113406637/192745656-755d76ea-4398-4e31-b2b1-94c4ad9d2a51.png)
create role is same error
![image](https://user-images.githubusercontent.com/113406637/192956621-71ecfbf8-3019-4f99-b2c6-1b48a431e947.png)

**Comment by daviszhen** [comment]
it was fixed in the local. the code has not been submitted.

**Comment by daviszhen** [comment]
if was fixed at #5587
